<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QMainWindow" name="main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1344</width>
    <height>857</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Malus - Logbook Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="load_data_button">
        <property name="text">
         <string>Load data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="export_button">
        <property name="text">
         <string>Save graph as txt</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="selected_item">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Bias Voltage</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Coil 1 current</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Coil 2 current</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MDX 1 Power</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MDX 1 Current</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MDX 1 Voltage</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MDX 2 Power</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MDX 2 Current</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MDX 2 Voltage</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ar Flow (SCCM)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>N2 Flow (SCCM)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>value11</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>value12</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>value13</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>value14</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>value15</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>value16</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>value17</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>value18</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Delay per second</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Total delay</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Time vs ticks</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="graphlayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="actionOpen_SampleDB">
   <property name="text">
    <string>Open SampleDB</string>
   </property>
   <property name="shortcut">
    <string comment="test" extracomment="test"/>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOff_specular">
   <property name="text">
    <string>Off-specular</string>
   </property>
  </action>
  <action name="actionNeutrons">
   <property name="text">
    <string>Neutrons</string>
   </property>
  </action>
  <action name="actionX_Rays">
   <property name="text">
    <string>X-Rays</string>
   </property>
  </action>
  <action name="actionOpen_single_specular_file">
   <property name="text">
    <string>Open single specular file</string>
   </property>
  </action>
  <action name="actionSpecular_X_ray">
   <property name="text">
    <string>Specular X-ray</string>
   </property>
  </action>
  <action name="menuXraySource">
   <property name="text">
    <string>X-rays</string>
   </property>
  </action>
  <action name="menuNeutronSource">
   <property name="text">
    <string>Neutrons</string>
   </property>
  </action>
  <action name="actionSpecular_neutron">
   <property name="text">
    <string>Specular neutron</string>
   </property>
  </action>
  <action name="actionOff_specular_neutron">
   <property name="text">
    <string>Off-specular neutron</string>
   </property>
  </action>
  <action name="actionData_tools">
   <property name="text">
    <string>General analysis</string>
   </property>
  </action>
  <action name="actionDetect_peaks">
   <property name="text">
    <string>Detect peaks</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
